packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -25,30 +25,43 @@ import 'theme.dart';
 ///
 ///     assert(debugCheckHasMaterial(context));
 class InkResponse extends StatefulWidget {
+  /// Creates an area of a [Material] that responds to touch.
+  ///
+  /// Must have an ancestor [Material] widget in which to cause ink reactions.
   InkResponse({
     Key key,
     this.child,
     this.onTap,
     this.onDoubleTap,
     this.onLongPress,
     this.onHighlightChanged,
-    this.containedInWell: false,
+    this.containedInkWell: false,
```

Comment:
i missed this rename in my original review. The original term was "is this ink response contained inside the well". The new term seems to be "is this a contained ink well". I think the old term made more sense.